main.go

```diff
@@ -8,6 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
+	_ "github.com/rancherio/go-machine-service/handlers/providers/amazonec2"
```

Comment:
Since we're on the subject, I'm not a big fan of the implicit registration of handlers through each files init() function. I would much rather see main.go explicitly register each handler. This gives us a more flexible dependency-injection style framework.